util/types/datum.go

```diff
@@ -1166,6 +1170,40 @@ func (d *Datum) ToFloat64() (float64, error) {
 	}
 }
 
+// ToString converts an interface to a string.
```

Comment:
The comment is not clear.
"ToString gets the string representation of the datum" is better?